RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZT_vect.cc

```diff
@@ -414,8 +416,8 @@ double DAClusterizerInZT_vect::update(double beta, track_t& gtracks, vertex_t& g
     auto o_trk_err_t = tks_vec.dt2_ptr[track_num];
     auto tmp_trk_z = tks_vec.z_ptr[track_num];
     auto tmp_trk_t = tks_vec.t_ptr[track_num];
-
     // auto-vectorized
+#pragma GCC ivdep
```

Comment:
this is now the only remaining pragma added in this PR.
It is not present in the InZ_vect file.
Is it really helping only in this case?

Reply:
Yes.